doc/designs/index.rst

```diff
@@ -7,6 +7,7 @@ FreeIPA design documentation
    adtrust/oneway-trust-with-shared-secret.md
    adtrust/samba-domain-controller.md
    adtrust/samba-domain-member.md
+   black-codestyle
```

Comment:
I think you want that as black-codestyle.md

Reply:
No, it's a RST file and RST files don't need a file extension when including in a TOC tree.

Reply:
Oy, ok